parm/config/gefs/config.efcs

```diff
@@ -38,26 +38,22 @@ if [[ "${RUN}" == "enkfgfs" ]]; then
    export FHMAX=${FHMAX_ENKF_GFS:-${FHMAX}}
 fi
 
-# Use serial I/O for ensemble (lustre?)
-export OUTPUT_FILETYPE_ATM="netcdf"
-export OUTPUT_FILETYPE_SFC="netcdf"
-
 # Stochastic physics parameters (only for ensemble forecasts)
 export DO_SKEB="YES"
-export SKEB=0.3
-export SKEB_TAU=21600.
-export SKEB_LSCALE=250000.
-export SKEBNORM=0
+export SKEB="0.8,-999,-999,-999,-999"
```

Comment:
I am not sure what stochastic physics setting GDAS will use? as this efcs will also be used by GDAS, it will be better to ask Cathy about this?

Reply:
As of now, we won't be changing the stochastic physics settings for the GDAS.  This config file won't be shared by GDAS though.  Our config files are in [parm/config/gfs](https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/gfs/config.efcs), so you can change this one without affecting us.